.circleci/config.yml

```diff
@@ -72,6 +72,25 @@ jobs:
       - store_test_results:
           path: /go/out/tests
 
+  nightly:
+    <<: *integrationDefaults
```

Comment:
looks like this needs to be `*defaults` to use the docker image with helm so `make lint` will work.  We should probably just make this `nightly` job the docker push job and add the `lint` and `install-cni` jobs to the "nightly" workflow section.

Reply:
Move lint to separate job in the workflow.

I'll iterate - there are few other optimizations we can make, first step is to get things moving.

Reply:
sounds good